cognite/client/_api/datapoints.py

```diff
@@ -451,7 +451,7 @@ def _add_to_subtask_pools(self, new_subtasks: Iterable[BaseDpsFetchSubtask]) ->
     def _queue_new_subtasks(
         self, pool: PriorityThreadPoolExecutor, futures_dct: Dict[Future, List[BaseDpsFetchSubtask]]
     ) -> None:
-        while pool._work_queue.qsize() == 0 and any(self.subtask_pools):
+        while pool._work_queue.empty() and any(self.subtask_pools):
```

Comment:
Interesting that these don't agree

Reply:
This was just me doing a bit of code golfing 😅